neuraxle/steps/sklearn.py

```diff
@@ -35,13 +35,14 @@ def __init__(
             self,
             wrapped_sklearn_predictor,
             hyperparams_space: HyperparameterSpace = None,
-            return_all_sklearn_default_params_on_get=False
+            return_all_sklearn_default_params_on_get=False,
+            hasher: Hasher = IDHasher()
```

Comment:
I wouldn't specify a hasher here and I'd let the BaseStep receive None to create an `HasherByHyperparameter`. As in my firsts comment way above.